pkg/operator/controller/ingress/controller.go

```diff
@@ -561,8 +561,29 @@ func setDefaultPublishingStrategy(ic *operatorv1.IngressController, platformStat
 					statusLB.ProviderParameters.GCP.ClientAccess = specClientAccess
 					changed = true
 				}
+			case operatorv1.IBMLoadBalancerProvider:
+				// The only provider parameter that is supported
+				// for IBM is the Protocol parameter.
+				var specProtocol, statusProtocol operatorv1.IngressControllerProtocol
+				if specLB.ProviderParameters != nil && specLB.ProviderParameters.IBM != nil {
+					specProtocol = specLB.ProviderParameters.IBM.Protocol
+				}
+				if statusLB.ProviderParameters != nil && statusLB.ProviderParameters.IBM != nil {
+					statusProtocol = statusLB.ProviderParameters.IBM.Protocol
+				}
+				if specProtocol != statusProtocol {
+					if statusLB.ProviderParameters == nil {
+						statusLB.ProviderParameters = &operatorv1.ProviderLoadBalancerParameters{
+							Type: operatorv1.IBMLoadBalancerProvider,
+						}
+					}
+					if statusLB.ProviderParameters.IBM == nil {
+						statusLB.ProviderParameters.IBM = &operatorv1.IBMLoadBalancerParameters{}
+					}
+					statusLB.ProviderParameters.IBM.Protocol = specProtocol
+					changed = true
+				}
```

Comment:
Follow the style we use in the other cases in order to reduce nesting:
```suggestion
				if specProtocol != statusProtocol {
					if statusLB.ProviderParameters == nil {
						statusLB.ProviderParameters = &operatorv1.ProviderLoadBalancerParameters{}
					}
					if len(statusLB.ProviderParameters.Type) == 0 {
							statusLB.ProviderParameters.Type = operatorv1.IBMLoadBalancerProvider
					}
					if statusLB.ProviderParameters.IBM == nil {
						statusLB.ProviderParameters.IBM = &operatorv1.IBMLoadBalancerParameters{}
					}
					statusLB.ProviderParameters.IBM.Protocol = specProtocol
					changed = true
				}
```